aries_vcx/src/errors/mapping_others.rs

```diff
@@ -21,3 +23,145 @@ impl<T> From<PoisonError<T>> for AriesVcxError {
+            AriesVcxErrorKind::PoolLedgerConnect => AriesVcxCoreErrorKind::PoolLedgerConnect,
+            AriesVcxErrorKind::InvalidLedgerResponse => AriesVcxCoreErrorKind::InvalidLedgerResponse,
+            AriesVcxErrorKind::LedgerItemNotFound => AriesVcxCoreErrorKind::LedgerItemNotFound,
+            AriesVcxErrorKind::NoPoolOpen => AriesVcxCoreErrorKind::NoPoolOpen,
+            AriesVcxErrorKind::PostMessageFailed => AriesVcxCoreErrorKind::PostMessageFailed,
+            AriesVcxErrorKind::WalletCreate => AriesVcxCoreErrorKind::WalletCreate,
+            AriesVcxErrorKind::WalletAccessFailed => AriesVcxCoreErrorKind::WalletAccessFailed,
+            AriesVcxErrorKind::InvalidWalletHandle => AriesVcxCoreErrorKind::InvalidWalletHandle,
+            AriesVcxErrorKind::DuplicationWallet => AriesVcxCoreErrorKind::DuplicationWallet,
+            AriesVcxErrorKind::WalletRecordNotFound => AriesVcxCoreErrorKind::WalletRecordNotFound,
+            AriesVcxErrorKind::DuplicationWalletRecord => AriesVcxCoreErrorKind::DuplicationWalletRecord,
+            AriesVcxErrorKind::WalletNotFound => AriesVcxCoreErrorKind::WalletNotFound,
+            AriesVcxErrorKind::WalletAlreadyOpen => AriesVcxCoreErrorKind::WalletAlreadyOpen,
+            AriesVcxErrorKind::DuplicationMasterSecret => AriesVcxCoreErrorKind::DuplicationMasterSecret,
+            AriesVcxErrorKind::DuplicationDid => AriesVcxCoreErrorKind::DuplicationDid,
+            AriesVcxErrorKind::LoggingError => AriesVcxCoreErrorKind::LoggingError,
+            AriesVcxErrorKind::EncodeError => AriesVcxCoreErrorKind::EncodeError,
+            AriesVcxErrorKind::UnknownError => AriesVcxCoreErrorKind::UnknownError,
+            AriesVcxErrorKind::InvalidDid => AriesVcxCoreErrorKind::InvalidDid,
+            AriesVcxErrorKind::InvalidVerkey => AriesVcxCoreErrorKind::InvalidVerkey,
+            AriesVcxErrorKind::InvalidNonce => AriesVcxCoreErrorKind::InvalidNonce,
+            AriesVcxErrorKind::InvalidUrl => AriesVcxCoreErrorKind::InvalidUrl,
+            AriesVcxErrorKind::SerializationError => AriesVcxCoreErrorKind::SerializationError,
+            AriesVcxErrorKind::NotBase58 => AriesVcxCoreErrorKind::NotBase58,
+            AriesVcxErrorKind::ParsingError => AriesVcxCoreErrorKind::ParsingError,
+            AriesVcxErrorKind::InvalidHttpResponse => AriesVcxCoreErrorKind::InvalidHttpResponse,
+            AriesVcxErrorKind::InvalidMessages => AriesVcxCoreErrorKind::InvalidMessages,
+            AriesVcxErrorKind::VdrToolsError(u32) => AriesVcxCoreErrorKind::VdrToolsError(u32),
+            AriesVcxErrorKind::UrsaError => AriesVcxCoreErrorKind::UrsaError,
+            AriesVcxErrorKind::NoAgentInformation => AriesVcxCoreErrorKind::NoAgentInformation,
+            AriesVcxErrorKind::InvalidMessageFormat => AriesVcxCoreErrorKind::InvalidMessageFormat,
+        };
+        AriesVcxCoreError::from_msg(kind, format!("AriesVcxError: {}", err.to_string()))
+    }
+}
```

Comment:
I bet you were annoyed having to do this 😆 .

We can do better in terms of error handling, but it's definitely outside the scope of this PR as it affects the entire workspace.